### PR TITLE
Select component

### PR DESCRIPTION
**Pull request by sorja**
Example usage:

image

### CODE REVIEW

File: src/client/components/Select/Select.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@import 'src/client/style/partials';
+
```
Comment (minotogna) [inline src/client/components/Select/Select.scss]: can we put the Select inside components/Inputs folder ?

File: src/client/components/Select/Select.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+.fra-select {
```
Comment (minotogna) [inline src/client/components/Select/Select.scss]: can we avoid fra prefix ?

File: src/client/components/Select/Select.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      return item.value === value
+    })
+  }, [options, value])
+
+  const _onChange = useCallback(
+    (option: TranslatedOption) => {
+      onChange(option?.value ?? null)
+    },
+    [onChange]
+  )
```
Comment (minotogna) [inline src/client/components/Select/Select.tsx]: can we have a hook called useValues that returns {defaultValue, options} ?

File: src/client/components/Select/Select.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import { Option } from './types'
+
+type Props = {
+  value: string
+  onChange: (newValue: string) => void
+  disabled: boolean
+  options: Array<Option>
+}
+
+type TranslatedOption = {
```
Comment (minotogna) [inline src/client/components/Select/Select.tsx]: does ReactSelect have its own type for this? I'm not a big fan of TranslatedOption type.
Also, TranslatedOptions type is not really needed.

Comment (sorja) [reply src/client/components/Select/Select.tsx]: I agree, I don't like the TranslatedOptions also, I tried to look in to the docs ( https://react-select.com/typescript ) and migration docs ( https://react-select.com/upgrade )


OptionTypeBase | no replacement | Options can be any type (if using getOptionValue and getOptionLabel) so there's no longer a base type for options
-- | -- | --


---

I liked the idea of passing `Label` type to the Select component, maybe we have to pass directly the translated instead?

**Pull request closed without merging**